tests/phpunit/includes/plural-form-function.php

```diff
@@ -7,11 +7,13 @@
  * @param string $expression
  */
 function tests_make_plural_form_function( $nplurals, $expression ) {
-	$expression = str_replace( 'n', '$n', $expression );
-	$func_body  = "
-		\$index = (int)($expression);
-		return (\$index < $nplurals)? \$index : $nplurals - 1;";
+	$closure = function ( $n ) use ( $nplurals, $expression ) {
+		$expression = str_replace( 'n', $n, $expression );
 
-	// phpcs:ignore WordPress.PHP.RestrictedPHPFunctions.create_function_create_function
-	return create_function( '$n', $func_body );
+		// phpcs:ignore Squiz.PHP.Eval -- This is test code, not production.
+		$index = (int) eval( 'return ' . $expression . ';' );
+		return ( $index < $nplurals ) ? $index : $nplurals - 1;
```

Comment:
Wouldn't it be cleaner to `return` right away, and also make it a static closure?
```php
return static function ( $n ) use ( $nplurals, $expression ) {
		$expression = str_replace( 'n', $n, $expression );

		// phpcs:ignore Squiz.PHP.Eval -- This is test code, not production.
		$index = (int) eval( 'return ' . $expression . ';' );
		return ( $index < $nplurals ) ? $index : $nplurals - 1;
	};
```

Reply:
I agree this should have been a `static` closure.

Regarding return straight away - yes, that would be possible, but I don't think that makes the code more readable.